src/Panel/Panel/Panel.less

```diff
@@ -1,12 +1,10 @@
-@import '../../style/variables.less';
-
-@handlestyle: 5px solid fade(@primary-color, 50%);
+@handlestyle: 5px solid var(--ant-primary-4);
 
 .react-geo-panel {
   z-index: 100;
 
   .body {
-    background-color: @component-background;
+    background-color: white;
```

Comment:
Is there no replacement for `@component-background`?

Reply:
No, only a small subset of the existing less variables is available yet.